spec/negotiation.html

```diff
@@ -94,6 +94,10 @@ <h1>
           1. If _pos_ &ge; 2 and the character *"-"* occurs at index _pos_ - 2 of candidate, decrease _pos_ by 2.
           1. Let _candidate_ be the substring of _candidate_ from position 0, inclusive, to position _pos_, exclusive.
       </emu-alg>
+
+      <emu-note>
+        In cases where Unicode T Extensions are included in _locale_, this algorithm may temporarily generate invalid language tags. However, it is not possible in any case for the abstract operation to return invalid language tags.
```

Comment:
```suggestion
        When _locale_ includes a <a href="https://unicode.org/reports/tr35/#BCP47_T_Extension">Unicode Technical Standard #35 Part 1 Core BCP 47 T Extension</a> subtag sequence, the truncation in this algorithm may temporarily generate invalid language tags. However, none of them will be returned because _availableLocales_ contains only valid language tags.
```